proto/beacon/p2p/v1/types.proto

```diff
@@ -116,4 +162,10 @@ message DepositMessage {
 
   // Deposit amount in gwei.
   uint64 amount = 3;
-}
\ No newline at end of file
+}
+
+// SyncCommittee serves as committees to facilitate light client syncing to beacon chain.
+message SyncCommittee {
+  repeated bytes pubkeys = 1  [(gogoproto.moretags) = "ssz-size:\"1024,48\""];
+  repeated bytes pubkey_aggregates = 2  [(gogoproto.moretags) = "ssz-size:\"16,48\""];
```

Comment:
What does the `48` part do in `ssz-size`?

Reply:
it tells SSZ that this should be treated as a [16][48]byte and not as a list